runtime/makelib/targets.mk.aix.inc.ftl

```diff
@@ -75,6 +78,11 @@ ifdef j9vm_uma_supportsIpv6
   CPPFLAGS += -DIPv6_FUNCTION_SUPPORT
 endif
 
+ifdef j9vm_uma_gnuDebugSymbols
+CFLAGS+=-g -qsuppress=1540-0825
+CXXFLAGS+=-g -qsuppress=1540-0825
```

Comment:
I'd prefer to see the dependencies fixed so we don't have to suppress those warnings.